app/src/main/java/org/p2p/wallet/infrastructure/network/interceptor/RpcInterceptor.kt

```diff
@@ -30,6 +50,60 @@ class ServerErrorInterceptor(
         }
     }
 
+    private fun createRpcRequest(chain: Interceptor.Chain): Request {
+        val request = chain.request()
+
+        val json = getRequestJson(request)
+
+        val url = request.url
+
+        val key = RpcConstants.REQUEST_METHOD_KEY
+        val value = RpcConstants.REQUEST_METHOD_VALUE_GET_CONFIRMED_TRANSACTIONS
+        val environmentUrl =
+            if (json?.getString(key) == value) {
+                Environment.RPC_POOL
+            } else {
+                currentEnvironment
+            }
```

Comment:
You may remove brackets here

```suggestion
            if (json?.getString(key) == value) Environment.RPC_POOL
            else currentEnvironment
```